### PR TITLE
Update exporting.rst

### PR DESCRIPTION
**Pull request by corrigentia**

### CODE REVIEW

File: getting_started/step_by_step/exporting.rst [hunk 1]
```diff
@@ -213,7 +213,7 @@ your system and the location of the keystore you just created.
 
 .. image:: img/export_editor_android_settings.png
 
-Now you're ready to export. Click on *Project -> Export* and add a preset
+Now, you're ready to export. Click on *Project -> Export* and add a preset
```
Comment (cbscribe) [inline getting_started/step_by_step/exporting.rst]: I'm sorry but this one I object to.  "Now you're ready" is a perfectly fine statement. This comma makes the sentence worse.

Ref:  "Now is the winter of our discontent..."

Comment (corrigentia) [reply getting_started/step_by_step/exporting.rst]: @cbscribe : "Now is the winter of our discontent [...]" (Richard III ... John Steinbeck's eponymous novel) -> now = noun. the present moment or time: "Now is the time to go."

"Click on Editor -> Editor Settings in Godot and select the Export/Android section. Here, you need to set the paths to the Android SDK applications on your system and the location of the keystore you just created.

Now, you’re ready to export." -> now = adverb. 8.b. Used to indicate a change of subject: "Now, let's get down to work."
10.a. used as a transitional particle.

"I'm sorry, but [...]" and "[...] but I object to this one." or "[...] but this is one I object to."
The latter is only required if you follow proper British English grammar rules.
to = preposition ("preposition" = [something] put in front [of something else])
Seeing as how you invoked William Shakespeare... But that might be asking too much of American English.

Either way, I appreciate this chance to practice applied grammar. Thank you for challenging me. I definitely wouldn't want to make something worse. I'd much rather see slow progress. Cheers! :smile:

Comment (akien-mga) [reply getting_started/step_by_step/exporting.rst]: Nobody likes a smart ass, you know?

Next time you quote someone else in an attempt to prove that you're better at English than they are, I will outright decline all your contributions. And yes you can also make fun of my English if you wish, I'm not a native speaker.

Discussing grammar is fine, but attacking people with statements such as "But that might be asking too much of American English." is not.

**Comment by akien-mga** [comment]
Merged manually as 213380f, thanks. (Minus the "Now**,** you're ready" change which is controversial - I also agree that it seems overkill).

File: getting_started/step_by_step/exporting.rst (hunk 1 above)
Comment (corrigentia) [reply getting_started/step_by_step/exporting.rst]: @cbscribe : I apologize if I've come across as an intellectual snob. I was referring to this being a case of "two nations divided by a common language". :gb: 

@akien-mga : That wasn't a personal attack. If anything, that might be construed as a cultural attack. Look into [John Taylor Gatto](https://en.wikipedia.org/wiki/John_Taylor_Gatto) and the [trivium](https://en.wikipedia.org/wiki/Trivium). I most certainly wasn't trying to use that as an argumentun ad hominem. I'm sorry if I've said or done anything that might make you feel like you're at risk of me making fun of you. I promise the thought has never crossed my mind. :innocent:

**Pull request closed without merging**